### DISCUSSION TITLE
丸辣，我就是冲冲浪，怎么这么晚了

### DISCUSSION THREAD

**Discussion by lzxixizi** [share121/inter-knot#1200] [其他]
我只不过是睡前打开了某个黑色的网站，随便翻了一下就这么晚了！
明天早上我还有跟派派的约会呢，求助网友有没有4个小时睡够8小时的办法

**Comment by lzxixizi** [comment]
事已至此，要么把每日委托做了吧

**Comment by fairy-B** [comment]
把你亲爱的**玉腿**垫高（光是小腿），和大腿、身子成90度直角，然后睡觉

**Comment by 1513298** [comment]
建议看闪老师【【干货】如何3小时睡够8小时？】 https://www.bilibili.com/video/BV12U411Z7A1/?share_source=copy_web

**Reply by 10kaihua** [reply]
非常有用的建议，楼主一定要看看
